src/main/java/com/okta/sdk/models/sessions/Session.java

```diff
@@ -1,9 +1,44 @@
 package com.okta.sdk.models.sessions;
 
+import com.fasterxml.jackson.annotation.JsonValue;
 import com.okta.sdk.framework.ApiObject;
+import com.okta.sdk.models.identityproviders.IdpObject;
+import org.joda.time.DateTime;
+
+import java.util.List;
 
 public class Session extends ApiObject {
 
+    public enum Status {
+        ACTIVE,         //The session is active
+        MFA_ENROLL,     //The user credentials are valid, but MFA is required and no factors are set up yet
+        MFA_REQUIRED;   //The user credentials are valid, but MFA is required
+    }
+
+    public enum AMR {
```

Comment:
can you rename this enum to something more descriptive or add a comment to describe it?